messagingclient/client.py

```diff
@@ -48,3 +53,87 @@ def callback_wrapper(payload):
                 # terminate on any exception so that the worker isn't hung.
                 future.cancel()
                 print(f"stopped listening: {exc}")
+
+    def consume_multiple(self, queues: Union[str, List], callback: Callable = None, max_messages: int = 1):
+        '''
+        TODO: This function can completely replace consume() (i.e. be renamed consume(), deleting the older version),
+        but for development, I left the original consume() untouched and added this as a new function instead.
+        To avoid altering any code that currently calls consume(), the existing version could eventually be deleted
+        and this version could be renamed in its place.
+        
+        If one queue is provided, it will be consumed until the process is terminated.
+        If multiple queues are provided, they will be consumed in round robin fashion.
+
+        Refs:
+        https://cloud.google.com/pubsub/docs/pull
+        '''
+        print("MessagingClient.consume_multiple()")
+        print("MessagingClient.consume_multiple() queues:", queues)
```

Comment:
Do we need all these extra print statements?

Reply:
No. I can clean them up.